.github/workflows/tests.yml

```diff
@@ -50,7 +51,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: ["3.9", "3.11"]
+        python-version: ["3.9", "3.10", "3.11"]
```

Comment:
Why was 3.10 not included here, previously? :thinking:

Reply:
So it was previously "3.8" and "3.10", and then it got changed to "3.9" and "3.11". I don't see why "3.10" shouldn't be included so I added it back in.